### PR TITLE
Zebra volume & tone setup added

### PR DESCRIPTION
**Pull request by Richard1793**
Added volume and tone setup to the zebra setup page

Index page:
- Zebra set up open in new page
- Added "Zebra QR Scanners Setup" to background information.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -410,7 +415,7 @@ <h3>What should the .csv file Look Like?</h3>
                 reader.onload = function(e) {
                     csvResult = e.target.result;
                     csvResult = $.csv.toObjects(csvResult);
-                    keys = identifyCSVKeys(csvResult);
+                    keys = identifyCSVKeys(csvResult, dose);
```
Comment (DrMikeyS) [inline index.html:418]: IS there any reason this change is needed for the zebra audio pull?

**Review by DrMikeyS — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Richard1793) [reply index.html:418]: You're right, that shouldn't be there.  The correct line is in the hybrid fix, ie that line should say:

` keys = identifyCSVKeys(csvResult, doseNumber);
`

Comment (Richard1793) [reply index.html:418]: Actually, for this change in isolation the line would be keys = identifyCSVKeys(csvResult)

Do I make that change and resubmit or do you do that by not accepting the change to that line?

Comment (Richard1793) [reply index.html]: I think I've committed a change that resolves this.  Fingers crossed ...

**Pull request merged**